docs/node-operators/cli.md

```diff
@@ -194,44 +467,251 @@ bye
+
+``` tab="cargo"
+cargo run -- node send --address=twit1ulyzvnknjnndkfva636erkkp83wxhhwdfhptsr --value=1234 --fee=1
 {"jsonrpc":"2.0","result":"a6c26804cf76e08ab379ea63e4aa046095dade2ae52fb3ecac90817583e61349","id":"1"}
 ```
 
 On error, returns the error message:
 
+``` tab="docker"
+docker exec -i witnet_node ./witnet node send --address=twit1ulyzvnknjnndkfva636erkkp83wxhhwdfhptsr --value=9999999999999999999 --fee=1
+{"jsonrpc":"2.0","error":{"code":-32603,"message":"Cannot build a transaction transferring more value than the current available balance: 9999999999999999999 + 1 > 39649999949502"},"id":"1"}
+```
+
+``` tab="binary"
+witnet node send --address=twit1ulyzvnknjnndkfva636erkkp83wxhhwdfhptsr --value=9999999999999999999 --fee=1
+{"jsonrpc":"2.0","error":{"code":-32603,"message":"Cannot build a transaction transferring more value than the current available balance: 9999999999999999999 + 1 > 39649999949502"},"id":"1"}
 ```
-witnet node send --pkh=twit1ulyzvnknjnndkfva636erkkp83wxhhwdfhptsr --value=9999999999999999999 --fee=1
+
+``` tab="cargo"
+cargo run -- node send --address=twit1ulyzvnknjnndkfva636erkkp83wxhhwdfhptsr --value=9999999999999999999 --fee=1
 {"jsonrpc":"2.0","error":{"code":-32603,"message":"Cannot build a transaction transferring more value than the current available balance: 9999999999999999999 + 1 > 39649999949502"},"id":"1"}
 ```
 
 Example with time lock set to 2019-10-01
 
+``` tab="docker"
+docker exec -i witnet_node ./witnet node send --address=twit1ulyzvnknjnndkfva636erkkp83wxhhwdfhptsr --value=1234 --fee=1 --time-lock=1569888000
+{"jsonrpc":"2.0","result":"e5b55ec4930f32383e63de9316238f369ee26d89d4375521071a885fc46b4c17","id":"1"}
 ```
-witnet node send --pkh=twit1ulyzvnknjnndkfva636erkkp83wxhhwdfhptsr --value=1234 --fee=1 --time-lock=1569888000
+
+``` tab="binary"
+witnet node send --address=twit1ulyzvnknjnndkfva636erkkp83wxhhwdfhptsr --value=1234 --fee=1 --time-lock=1569888000
 {"jsonrpc":"2.0","result":"e5b55ec4930f32383e63de9316238f369ee26d89d4375521071a885fc46b4c17","id":"1"}
 ```
 
+``` tab="cargo"
+cargo run -- node send --address=twit1ulyzvnknjnndkfva636erkkp83wxhhwdfhptsr --value=1234 --fee=1 --time-lock=1569888000
+{"jsonrpc":"2.0","result":"e5b55ec4930f32383e63de9316238f369ee26d89d4375521071a885fc46b4c17","id":"1"}
+```
+
+### send-request
+
+Send a serialized data request.
+
+```sh tab="docker"
+docker exec -i witnet_node ./witnet node send-request 33c656101dd1dfa2258415d6487c648152b95983d1151f46a679b5d1902f6a8d
+```
+
+```sh tab="cargo"
+cargo run -- node send-request 33c656101dd1dfa2258415d6487c648152b95983d1151f46a679b5d1902f6a8d
+```
+
+```sh tab="binary"
+witnet node send-request 33c656101dd1dfa2258415d6487c648152b95983d1151f46a679b5d1902f6a8d
+```
+
+### server
+
+Run a Witnet node server.
+
+```sh tab="docker"
+docker exec -i witnet_node ./witnet node server
+```
+
+```sh tab="binary"
+witnet node server
+```
+
+```sh tab="cargo"
+cargo run -- node server
+```
+
+### show-config
```

Comment:
This is not implemented, if we want it implemented let's open an issue in witnet-rust